src/lecturer/services/StoreService.tsx

```diff
@@ -0,0 +1,64 @@
+import { createContext, ReactNode, useState } from "react";
```

Comment:
is this file some sort of a broker/container to store quizes for other functions?

Reply:
The store is using react context to provide objects to all components inside him. 
https://pl.reactjs.org/docs/context.html